hack/go-mod.sh

```
@@ -7,3 +7,8 @@ cd ./tools
 go mod tidy
 go mod vendor
 go mod verify
+
+cd ../openshift-tests-extension
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)